.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -32,7 +32,7 @@ Other classifications:
 
 > 0.2.x
 > 0.3.x
-> HEAD
+> Git SHA
```

Comment:
```suggestion
> Git SHA (Please provide SHA output from: `git rev-parse HEAD`)
```

It is fine as is, but might be helpful to provide the command to easily get the current sha